internal/rpchelp/helpdescs_en_US.go

```diff
@@ -0,0 +1,399 @@
+	"listtransactionsresult-category":        `The kind of transaction: "send" for sent transactions, "immature" for immature coinbase outputs, "generate" for mature coinbase outputs, or "recv" for all other received outputs.  Note: A single output may be included multiple times under different categories`,
+	"listtransactionsresult-amount":          "The value of the transaction output valued in bitcoin",
+	"listtransactionsresult-fee":             "The total input value minus the total output value for sent transactions",
+	"listtransactionsresult-confirmations":   "The number of block confirmations of the transaction",
+	"listtransactionsresult-generated":       "Whether the transaction output is a coinbase output",
+	"listtransactionsresult-blockhash":       "The hash of the block this transaction is mined in, or the empty string if unmined",
+	"listtransactionsresult-blockindex":      "Unset",
+	"listtransactionsresult-blocktime":       "The Unix time of the block header this transaction is mined in, or 0 if unmined",
+	"listtransactionsresult-txid":            "The hash of the transaction",
+	"listtransactionsresult-walletconflicts": "Unset",
+	"listtransactionsresult-time":            "The earliest Unix time this transaction was known to exist",
+	"listtransactionsresult-timereceived":    "The earliest Unix time this transaction was known to exist",
+	"listtransactionsresult-comment":         "Unset",
+	"listtransactionsresult-otheraccount":    "Unset",
+
+	// ListTransactionsCmd help.
+	"listtransactions--synopsis":        "Returns a JSON array of objects containing verbose details for wallet transactions.",
+	"listtransactions-account":          "DEPRECATED -- Unused",
+	"listtransactions-count":            "Maximum number of transactions to create results from",
+	"listtransactions-from":             "Number of transactions to skip before results are created",
+	"listtransactions-includewatchonly": "Unused",
+
+	// ListUnspentCmd help.
+	"listunspent--synopsis": "Returns a JSON array of objects representing unlocked unspent outputs controlled by wallet keys.",
+	"listunspent-minconf":   "Minimum number of block confirmations required before a transaction output is considered",
+	"listunspent-maxconf":   "Maximum number of block confirmations required before a transaction output is excluded",
+	"listunspent-addresses": "If set, limits the returned details to unspent outputs received by any of these payment addresses",
+
+	// ListUnspentResult help.
+	"listunspentresult-txid":          "The transaction hash of the referenced output",
+	"listunspentresult-vout":          "The output index of the referenced output",
+	"listunspentresult-address":       "The payment address that received the output",
+	"listunspentresult-account":       "The account associated with the receiving payment address",
+	"listunspentresult-scriptPubKey":  "The output script encoded as a hexadecimal string",
+	"listunspentresult-redeemScript":  "Unset",
```

Comment:
For compatibility, yes.  For just the btcjson conversion, no.

It's also saved in the db with the script crypto key and would require the entire wallet to be unlocked to access it.